src/App.js

```diff
@@ -266,110 +266,119 @@ class App extends Component {
-                 geojson={this.state.geojson}
-                 user={this.state.user}
-                 config={this.props.config}
-                 embeddable={this.props.history.location.pathname === this.props.config.PAGES.embeddable.path}
-            />
-
-            <Dehaze className={classes.burger} onClick={this.toggleLeftDrawer(true)}
-              style={{
-                display: this.state.welcomeShown && this.props.history.location.pathname === this.props.config.PAGES.map.path
+              }
+
+              <Route path={this.props.config.PAGES.writeFeedback.path} render={(props) =>
+                <WriteFeedbackPage {...props}
+                  user={this.state.user}
+                  location={this.state.location}
+                  online={this.state.online}
+                  handleClose={this.goToMap}
+                  label={this.props.config.PAGES.writeFeedback.label}
+                />}
+              />
+
+            </Switch>
+          }
+
+          {!this.state.welcomeShown && this.props.history.location.pathname !== this.props.config.PAGES.embeddable.path &&
+            this.state.termsAccepted &&
+            <WelcomePage handleClose={this.handleWelcomePageClose} />
+          }
+
+          <Map location={this.state.location}
+            visible={[this.props.config.PAGES.map.path, this.props.config.PAGES.embeddable.path].includes(this.props.history.location.pathname)}
+            welcomeShown={this.state.welcomeShown || this.props.history.location.pathname === this.props.config.PAGES.embeddable.path}
+            geojson={this.state.geojson}
+            user={this.state.user}
+            config={this.props.config}
+            embeddable={this.props.history.location.pathname === this.props.config.PAGES.embeddable.path}
+          />
+
+          <Dehaze className={classes.burger} onClick={this.toggleLeftDrawer(true)}
+            style={{
+              display: this.state.welcomeShown && this.props.history.location.pathname === this.props.config.PAGES.map.path
                 ? 'block'
                 : 'none'
-              }}
-            />
+            }}
+          />
 
-            <Fab className={classes.camera} color="secondary" onClick={this.handlePhotoClick}
-              style={{
-                display: this.state.welcomeShown && this.props.history.location.pathname === this.props.config.PAGES.map.path
+          <Fab className={classes.camera} color="secondary" onClick={this.handlePhotoClick}
+            style={{
+              display: this.state.welcomeShown && this.props.history.location.pathname === this.props.config.PAGES.map.path
                 ? 'flex'
-                : 'none'
-              }}
-            >
-              <AddAPhotoIcon />
-            </Fab>
-          </main>
+                : 'none',
+
+              borderRadius: '10px',
```

Comment:
1, I had a quick look and couldn't see a text prop here
<img width="323" alt="Screenshot 2019-07-16 at 19 51 19" src="https://user-images.githubusercontent.com/29929268/61323274-743e7b00-a807-11e9-8d21-eb97890ab559.png">